l10n_it_fatturapa/views/partner_view.xml

```diff
@@ -9,17 +9,16 @@
             <notebook position="inside">
             <page name="fatturapa" string="Electronic Invoice" groups="account.group_account_invoice">
                 <group name="fatturapa_group">
-                    <label for="electronic_invoice_subjected"/>
                     <field name="electronic_invoice_subjected"/>
-
-                    <group attrs="{'invisible': [('electronic_invoice_subjected', '=', False)]}">
+                    <field name="electronic_invoice_obliged_subject" attrs="{'invisible': [('electronic_invoice_subjected', '=', False)]}"/>
+                </group>
+                    <group attrs="{'invisible': [('electronic_invoice_obliged_subject', '=', False)]}">
                         <field name="ipa_code" placeholder="IPA123" attrs="{'invisible': [('is_pa','=', False)]}"/>
                         <field name="codice_destinatario" attrs="{'invisible': [('is_pa', '=', True)]}"/>
```

Comment:
@primes2h in caso di cliente estero per cui voglio mandare e-fattura al SDI, che quindi avrebbe `electronic_invoice_subjected` = `True` e `electronic_invoice_obliged_subject` = `False`, il campo `codice_destinatario` viene impostato a `XXXXXXX` anche se invisibile?

Reply:
Esatto, viene impostato a `XXXXXXX` tramite `onchange` quando nel partner viene selezionata la nazione.

Tra l'altro mi sono appena accorto che attualmente non viene gestito il caso di partner estero esistente che viene abilitato alla fatturazione elettronica, il campo codice destinatario risulta il predefinito `0000000`.

Adesso carico il fix.